### ISSUE TITLE
snap run olivia-test error: cannot find app "olivia-test" in "olivia-test"

### ISSUE THREAD

**Issue by lwille** [keshavbhatt/olivia#31]
I installed the snap using `sudo snap install olivia-test` but can't run it.

```
$ snap run olivia-test

error: cannot find app "olivia-test" in "olivia-test"
```

May be a duplicate of #21 (we'll never know :laughing:)

```
$ snap --version
snap    2.37.4-2.fc29
snapd   2.37.4-2.fc29
series  16
fedora  29
kernel  5.0.3-200.fc29.x86_64

$ lsb_release -a
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: Fedora
Description:    Fedora release 29 (Twenty Nine)
Release:        29
Codename:       TwentyNine
```

**Comment by keshavbhatt** [comment]
try this :
snap run olivia-test.olivia

**Comment by keshavbhatt** [comment]
Closes #31
